Documentation/admin/logging.md

```diff
@@ -0,0 +1,94 @@
+
+If you want to customize your Elasticsearch output configuration or look at examples using different storage destinations, see the [customizing log destination][customizing-log-destination] document.
+
+## Deploying Fluentd
+
+First create the `logging` namespace for all of our resources to live in:
+
+```
+$ kubectl create ns logging
+```
+
+Then setup all the service account and roles that Fluentd needs to query Kubernetes for metadata about the containers logs it's watching:
+
+```
+$ kubectl create -f https://coreos.com/tectonic/docs/latest/files/logging/fluentd-service-account.yaml
+$ kubectl create -f https://coreos.com/tectonic/docs/latest/files/logging/fluentd-role.yaml
+$ kubectl create -f https://coreos.com/tectonic/docs/latest/files/logging/fluentd-role-binding.yaml
+```
+
+Next deploy Fluentd:
+
+```
+$ kubectl create -f https://coreos.com/tectonic/docs/latest/files/logging/fluentd-configmap.yaml
+$ kubectl create -f https://coreos.com/tectonic/docs/latest/files/logging/fluentd-svc.yaml
+$ kubectl create -f https://coreos.com/tectonic/docs/latest/files/logging/fluentd-ds.yaml
+```
+
+Finally, watch for the pods to become ready:
+
+```
+$ kubectl get pods --watch --namespace logging
+```
+
+Once all the pods are ready, everything should be functioning. To double check, use `kubectl logs` on one of the pods listed above to make sure there aren't any errors, and that Fluentd is able to send logs to where you've configured.
+
+
+### (Optional) Deploy Prometheus to monitor Fluentd
+
+Tectonic includes the [promtheus-operator][prometheus-operator] in installations by default. This operator can be used to create additional instances of Prometheus to monitor your apps.
+
+If you wish to enable Prometheus monitoring of your Fluentd pods, run the following commands:
+
+```
+$ kubectl create -f https://coreos.com/tectonic/docs/latest/files/logging/monitoring/prometheus-fluentd-role-binding.yaml
+$ kubectl create -f https://coreos.com/tectonic/docs/latest/files/logging/monitoring/prometheus-fluentd-service-account.yaml
+$ kubectl create -f https://coreos.com/tectonic/docs/latest/files/logging/monitoring/prometheus-fluentd.yaml
+$ kubectl create -f https://coreos.com/tectonic/docs/latest/files/logging/monitoring/fluentd-prometheus-service-monitor.yaml
+```
+
+## Customization
+
+If you would like to make customizations to your logging setup, view our doc on [customizing Fluentd][customizing-fluentd].
+
+
+[fluentd-config]: ../files/logging/fluentd-configmap.yaml
+[fluentd-ds]: ../files/logging/fluentd-ds.yaml
+[configuring-credentials]: ../tutorials/first-app.md#configuring-credentials
+[logging-config-files]: https://github.com/coreos/tectonic-installer/tree/master/Documentation/files/logging
```

Comment:
This link needs to be updated to point to the `tectonic-docs` repo.

Reply:
Good catch, but -- I'm not 100% sure it shuld be updated, since the files beneath `files/logging` are straight yaml and don't really have a presentation form on coreos.com.

Reply:
I agree, this should remain an outgoing link to the github directory. What I meant was that the link needs to be updated to point at `tectonic-docs`, because after the move, `files/logging` won't exist under `tectonic-installer` anymore (as seen [here](https://github.com/coreos/tectonic-installer/pull/1810/files#diff-02cbd1636fe9a721a5619a564300e788)).